### DISCUSSION TITLE
收藏集/数字收藏 自定义顶栏问题

### DISCUSSION THREAD

**Discussion by CloudyEagle** [the1812/Bilibili-Evolved#5083] [综合讨论 / General]
浏览器打开控制台，并启用设备模拟的功能中设定成的手机型号后，就能用访问收藏集或者数字收藏的页面。
尽管在浏览器打开这个页面主要为了扒收藏集或者数字收藏的原图，无法正常操作页面；
但还是希望这页面不使用该脚本，因为跟人感觉看去有点怪怪的。
![image](https://github.com/user-attachments/assets/2dd8a310-9401-481e-a398-e5453e393ee0)

**Comment by the1812** [comment]
手机端的页面不会进行适配

**Reply by CloudyEagle** [reply]
不适配可以理解，可以做到不显示吗？还是您认为不显示也是一种适配。

**Reply by the1812** [reply]
不显示也是一种适配
